### ISSUE TITLE
Invalid API Response for peers endpoint with sort by height desc

### ISSUE THREAD

**Issue by ManuGowda** [LiskHQ/lisk-sdk#2861]
### Expected behavior
API `api/peers?sort=height:desc&limit=100` should give peers list by `DESC` `Height`
### Actual behavior
API `api/peers?sort=height:desc&limit=100` gives peers without height first insted of highest `height` peers.
```
{
"ip": "176.160.50.130",
"wsPort": 8001,
"state": 1,
"version": "1.3.0",
"nonce": "jgEqj0CsTtLCOOUs",
"httpPort": 8000
},
{
"ip": "94.237.51.25",
"wsPort": 8001,
"state": 2,
"os": "linux4.4.0-142-generic",
"version": "1.4.0",
"broadhash": "4c1ff319dbb4f6576ccb0803ab6c778fbc5d95127cc3e7700bf989e9d9f31292",
"height": 8334916,
"nonce": "REKhDHmJ34k8fxKv",
"httpPort": 8000
}
```
### Steps to reproduce
https://node.lisk.io/api/peers?sort=height:desc&limit=100

**Comment by ManuGowda** [comment]
Conversation From Pablo: The issue is that Peers.height can be null as per the code; my fix is forcing a height of 0 when height is undefined but don’t know if that’s valid. Could you confirm if such fix would be valid?

**Comment by ManuGowda** [comment]
@pablitovicente Hey Pablo, sorry for late replay, I think undefined will be invalid value anyways, regarding giving 0 value to it will need more discussion, because node might be dead or node might be at certain height and not responding or peer might not exists and the node still persists the peer and shows undefined.

so there are lot of possibilities to it, so we can’t assign 0 to it, we need evaluate why its undefined, is it dead, or not reachable, and based on that we need to come to conclusion.

**Comment by shuse2** [comment]
This should be solved by #3295

**Comment by shuse2** [comment]
Closed by #3295
